src/components/app/createRoutes.jsx

```diff
@@ -14,9 +15,12 @@ export default function createRoutes(routes, authHoc = null) {
       );
     }
 
+    // Either standard or universal component
+    const component = route.eager ? route.component : universal(route.component, route.async || {});
```

Comment:
@rsumilang we decided there was no promise check?  so no arrow function in route config to accept props if desired?